src/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "awesome-emoji-picker",
+  "version": "2.0.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rugk/awesome-emoji-picker.git"
+  },
+  "author": "rugk",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/rugk/awesome-emoji-picker/issues"
+  },
+  "homepage": "https://github.com/rugk/awesome-emoji-picker#readme",
+  "dependencies": {
+    "@emoji-mart/data": "^1.0.1",
+    "emoji-mart": "^5.0.0"
```

Comment:
TODO: Update to emoji-mart v5.1.0

https://github.com/missive/emoji-mart/releases/tag/v5.1.0